systers_portal/static/css/style.css

```diff
@@ -85,6 +86,11 @@ body > .container {
   margin: 0 auto;
 }
 
+.add_email {
```

Comment:
This approach works, but there is even a better one. Take a look at the [bootstrap grid system](http://getbootstrap.com/css/#grid). This is a 12-column responsive grid.

I suggest to wrap the content into a row and columns of 3 or 4 (see which ones look nicer). That makes the block responsive and we are not adding any additional classes. Moreover 350px is a fixed width, no matter what screen size you have. I really encourage you to learn about bootstrap grid system. If you have additional questions, find me on gitter.